cranelift/fuzzgen/src/config.rs

```diff
@@ -38,6 +42,7 @@ impl Default for Config {
             block_signature_params: 0..=16,
             jump_tables_per_function: 0..=4,
             jump_table_entries: 0..=16,
+            switch_target_blocks: 0..=16,
```

Comment:
I think increasing this a bit will allow for more interesting patterns of generated code.